eng/PublishRelease.ps1

```diff
@@ -0,0 +1,45 @@
+param($NpmToken, $GitHubToken, $BuildNumber, $Sha, $WorkingDirectory, $PackageJson, $CoverageUser, $CoveragePass)
+
+$WorkingDirectory = Resolve-Path $WorkingDirectory
+$RepoRoot = Resolve-Path "$PSScriptRoot/.."
+
+Copy-Item $PackageJson $WorkingDirectory -Force
+
+Push-Location $WorkingDirectory
+try {
+    $currentVersion = node -p -e "require('./package.json').version";
+    $devVersion = "$currentVersion-dev.$BuildNumber"
+
+    Write-Host "Setting version to $devVersion"
+
+    npm version --no-git-tag-version $devVersion | Out-Null;
+   
+    $file = npm pack -q;
+    $name = "Autorest for c# v$devVersion"
+
+    Write-Host "Publishing $file on GitHub!"
+    
+    cmd /c ""npx -q publish-release --token $GitHubToken --repo autorest.csharp --owner azure --name $name --tag v$devVersion --notes=prerelease-build --prerelease --editRelease false --assets $file --target_commitish $Sha 2>&1""
+
+    $filePath = Join-Path $WorkingDirectory '.npmrc'
```

Comment:
Should we use the recommedned pattern of checking in the npmrc and setting the token as an env var: https://docs.npmjs.com/using-private-packages-in-a-ci-cd-workflow#set-the-token-as-an-environment-variable-on-the-cicd-server

Reply:
I don't feel strongly.